gettext_i18n_rails.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new name, GettextI18nRails::VERSION do |s|
   s.add_development_dependency "haml"
   s.add_development_dependency "rake"
   s.add_development_dependency "rails"
-  s.add_development_dependency "ruby_parser", ">= 3"
+  s.add_development_dependency "ruby_parser", ">= 3.7.1"
```

Comment:
I don't quite understand what you mean? Shall I add the 'gem' call to the file that needs the parser? Meaning into `lib/gettext_i18n_rails/ruby_gettext_extractor.rb`?

The version is needed to support ruby 2.2. Older ruby works with older versions of the gem.

Please, try to reformulate what you want me to do. I'll do whatever is needed for sure ;-)